lib/downloader.rb

```diff
@@ -162,7 +162,10 @@ def external_downloader(uri, filename = File.basename(url), verbose = false)
   #      %<retry>: Will be substitute to #{CREW_DOWNLOADER_RETRY}
   #       %<url>s: Will be substitute to #{url}
   #    %<output>s: Will be substitute to #{filename}
-  curl_cmdline = 'curl %<verbose>s -L -# --retry %<retry>s %<url>s -o %<output>s'
+  # i686 curl throws a "SSL certificate problem: self signed certificate in certificate chain" error.
+  # Only bypass this when we are using the system curl early in install.
+  @default_curl = `which curl`.chomp
```

Comment:
this does not need to be an instance variable

Reply:
Doesn't hurt since it could potentially be used in another place.

Reply:
We shouldn't expand scope until we need to...

Reply:
We think differently.  There is absolutely no harm in doing this at all.

Reply:
Do you want to fix this in a follow-up?